homeassistant/components/dynalite/bridge.py

```diff
@@ -24,7 +25,9 @@
 class DynaliteBridge:
     """Manages a single Dynalite bridge."""
 
-    def __init__(self, hass: HomeAssistant, config: dict[str, Any]) -> None:
+    def __init__(
+        self, hass: HomeAssistant, config: dict[str, Any] | MappingProxyType[str, Any]
```

Comment:
It is still needed to accept a dictionary in this case?

Reply:
I assumed it, but did not test it....bad choice from my side, corrected.